homeassistant/components/almond/strings.json

```diff
@@ -1,15 +1,17 @@
 {
   "config": {
     "step": {
-      "pick_implementation": { "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]" },
+      "pick_implementation": {
+        "title": "[%key:common::config_flow::title::oauth2_pick_implementation%]"
+      },
       "hassio_confirm": {
         "title": "Almond via Hass.io add-on",
         "description": "Do you want to configure Home Assistant to connect to Almond provided by the Hass.io add-on: {addon}?"
       }
     },
     "abort": {
-      "already_setup": "You can only configure one Almond account.",
-      "cannot_connect": "Unable to connect to the Almond server.",
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
       "missing_configuration": "Please check the documentation on how to set up Almond.",
```

Comment:
The value for `missing_configuration` can be replaced with a reference to `oauth2_missing_configuration`.

Reply:
It is not used actually. Will remove it.

Reply:
It's used by our oauth2 helper, so it needs to stay:
https://github.com/home-assistant/core/blob/ee914366a8762f495a585e09a0bd7b0675b4011c/homeassistant/helpers/config_entry_oauth2_flow.py#L222

Reply:
Addressed here: https://github.com/home-assistant/core/pull/41390